pkg/controller/remotemachineset/remotemachineset_controller.go

```diff
@@ -128,6 +129,10 @@ func (r *ReconcileRemoteMachineSet) Reconcile(request reconcile.Request) (reconc
 		log.WithError(err).Error("error looking up cluster deployment")
 		return reconcile.Result{}, err
 	}
+	if cd.Annotations != nil && cd.Annotations[constants.AnnotationSyncsetPause] == "true" {
+		log.Debug("syncing to cluster is disabled")
```

Comment:
I would prefer this to be a info message as this information should be available in the logs easily as we do not want users to forget about the annotation and then wondering why syncsets are not working.

Reply:
Lets actually go Warn for this, it's a one off customization that should not be done and forgotten.

Reply:
Also lets mention in the message that `SyncsetPauseAnnotation is present, hence syncing to the cluster deployment is disabled`